starter_code/css/style.css

```diff
@@ -264,29 +236,21 @@ blockquote span {
 
 #bottom-cta {
   background: #f9e42e;
-  padding: 30px;
   text-align: center;
 }
 
 footer {
   background: #121212;
-  padding: 65px 0 45px;
   text-align: center;
 }
 footer .logo {
   color: #f9e42e;
-  margin-bottom: 68px;
 }
 footer ul li {
   display: inline-block;
   list-style: none;
-  margin: 0 26px 75px;
 }
 footer p {
-  font-size: 18px;
   color: #fff;
   font-weight: 600;
-}
-
-
-
+}
```

Comment:
The only thing that's missing is this:

```css
img {
  max-width: 100%;
}
```

This ruleset will prevent images from spilling outside their containers. This can be seen in some mobile view where the images wider than than the container